README.md

```diff
@@ -310,14 +310,14 @@ presented in [Müller2015](http://flyingmachinearena.org/wp-content/publications
   publisher={IEEE}
 }
 ```
-We adapted [https://github.com/markwmuller/RapidQuadrocopterTrajectories] to
+We adapted https://github.com/markwmuller/RapidQuadrocopterTrajectories to
```

Comment:
Why undo the link? [RapidQuadrocopterTrajectories](https://github.com/markwmuller/RapidQuadrocopterTrajectories) is my recommendation
```
[RapidQuadrocopterTrajectories](https://github.com/markwmuller/RapidQuadrocopterTrajectories) 
```